README.md

```diff
@@ -72,15 +72,15 @@ An image that has a pair of words is created. The answer to one of the words is
     - `id`: `String` - The uuid of the captcha generated
 
 
-### - `/media`
+### - `/media`: `POST`,`GET` 
```

Comment:
Is there a reason for `POST` support on this endpoint?

Reply:
I don't remember why exactly but it had something to do with the Wordpress plugin.

Reply:
Ok, I will create a separate issue for this.

Everything else looks fine.